tests/importance_tests/test_init.py

```diff
@@ -267,3 +269,44 @@ def objective(trial: Trial) -> float:
     assert all([param in param_importance for param in ["x", "y"]])
     assert param_importance["x"] == 1.0
     assert param_importance["y"] == 0.0
+
+
+def objective(trial: Trial) -> float:
+    x1 = trial.suggest_float("x1", 0.1, 3)
+    x2 = trial.suggest_float("x2", 0.1, 3, log=True)
+    x3 = trial.suggest_float("x3", 2, 4, log=True)
+    return x1 + x2 * x3
```

Comment:
How about renaming this function since it conflicts with inner function of `test_get_param_importances` and `test_get_param_importance_target_is_none_and_study_is_multi_obj`?

Reply:
Hmm... Good question 😄
I have no idea. We may define this function as an inner function like other test cases since these test cases checks different behavior.